pyaedt/icepak.py

```diff
@@ -6072,6 +6072,7 @@ def create_geometry_from_csv(self, csv_file, header_line=2, column_mapping=None,
             The default value is 2.
         column_mapping : dict, optional
             Map of the csv columns names to the accepted column names.
+            The default value is ``None``.
         geo_type: str, optional
             Type of geometry that the csv file describes. Accepted values are: ``"cylinder"``.
             Default is ``"cylinder"``.
```

Comment:
```suggestion
            Type of geometry that the CSV file describes. Accepted values are: ``"cylinder"``.
            The default is ``"cylinder"``.
```
What other values are acceptable other than "cylinder"? They need to be added to this sentence: Accepted values are: ``"cylinder"``. The verb "are" is plural and should introduce a list with multiple items.